### DISCUSSION TITLE
Black and White image has an Orange tint.

### DISCUSSION THREAD

**Discussion by 5C0U7MAN** [d2phap/ImageGlass#1878] [Q&A]
Hello, I drew something in Krita and it was drawn in black, white, and gray, but when I open the image in ImageGlass it has an orange tint. Cannot find where to change this or where the problem comes from. In the Windows image preview it is also black and white, only on IG is orange. Thanks.
![bw](https://github.com/d2phap/ImageGlass/assets/165170493/c1c3bdf6-37ef-4f77-8e14-58ada92880cc)
![bw-orange](https://github.com/d2phap/ImageGlass/assets/165170493/1cd98d27-f7ec-41b4-af89-b4fc3fc1456a)

**Comment by d2phap** [comment]
@5C0U7MAN can you send me the sample file?

**Reply by 5C0U7MAN** [reply]
hey, thanks for the response! do you mean the image itself? or is the sample file like the data about the image glass i got installed?
